src/Compiler/Service/TransparentCompiler.fs

```diff
@@ -1415,11 +1415,11 @@ type internal TransparentCompiler
                             if tcInfo.stateContainsNodes |> Set.contains fileNode then
                                 failwith $"Oops!"
 
-                            if
-                                tcInfo.stateContainsNodes
-                                |> Set.contains (NodeToTypeCheck.PhysicalFile(index + 1))
-                            then
-                                failwith $"Oops!!!"
+                            // if
```

Comment:
I'd say - either remove or comment :)

Reply:
Want me to remove the other [Oops??? in the PhysicalFile path](https://github.com/dotnet/fsharp/blob/f68d7b60cafcac940754774ee34d8cf8e90ab754/src/Compiler/Service/TransparentCompiler.fs#L1372-L1378) as well?

Reply:
Let's leave the comment, so we understand it was there in the first place, once we get to transparent compiler issues.